vignettes/Introduction.Rmd

```diff
@@ -583,7 +583,7 @@ a$fn()
 #> [1] 100
 ```
 
-As of R6 2.3.0, if the object is cloned, member (non-method) functions will not have their enclosing environment changed, which is what one would normally expect. It will behave this way:
+As of R6 2.5.0, if the object is cloned, member (non-method) functions will not have their enclosing environment changed, which is what one would normally expect. It will behave this way:
```

Comment:
Is this version correct?

Reply:
The latest CRAN version is `2.5.1`, and that's what I used to check this code, but I wasn't sure if you wanted to also have a specific patch version.

Reply:
I believe this is the issue it is referring to.
https://github.com/r-lib/R6/blob/ab020a50e0dc9a3d7dffa649116343a26683a96c/NEWS.md?plain=1#L43